firefox/its-a-tracker.html

```diff
@@ -0,0 +1,83 @@
+<!DOCTYPE HTML>
+<html lang="en-US" dir="ltr">
+<head>
+    <meta charset="utf-8">
+
+    <title>It's a tracker!</title>
+    <link href="/css/min.css" rel="stylesheet">
+    <link rel="stylesheet" href="/css/its-a-trap.css" media="screen" />
+
+</head>
+
+<body id="its-a-tracker" class="portal-page locale-en-US portal-page">
+
+<div id="wrapper">
+
+<div id="doc">
+
+    <div id="nav-access">
+        <a href="#nav-main">skip to Navigation</a>
+        <a href="#switch">switch language</a>
+    </div>
+
+    <!-- start #header -->
+    <div id="header">
+        <div>
+		<h1><a href="http://www.mozilla.org/firefox/" title="Back to home page">Mozilla Firefox</a></h1>
+		<a href="http://www.mozilla.org/" class="mozilla">visit <span>mozilla</span></a>
+
+        </div>
+    </div>
+    <!-- end #header -->
+
+
+<div id="main-feature">
+	<h2>It’s a Tracker!</h2>
+	<p>Boo! I’m a Web page full of trackers!</p>
+</div>
+
+<div id="main-content">
+
+<p>OK, not really, but Web trackers are out there on the Internet. This is a test page for Firefox's built-in <a href="https://support.mozilla.org/en-US/kb/tracking-protection-firefox">Tracking Protection</a>.</p>
```

Comment:
Nit, indentation, there are a lot if inconsistent indentations. Not a blocker but a style concern.